quick-start/data-demo/Kptfile

```diff
@@ -0,0 +1,70 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: data-demo
```

Comment:
How does data-demo differ from bucket demo / what's the purpose of this blueprint? We should have a README for all blueprints.